lib/noble_page.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+require_relative 'member_page'
+
+class NoblePage < MemberPage
+  field :constituency do
+    table_field('Constituency').match(' for (.*)').to_a[1]
```

Comment:
the equivalent field in MemberPage doesn't have the `to_a` here, which makes me wonder whether there's something subtly different about the two layouts. If there are, it would be worth noting in a comment. If not, it would be better to be consistent. (NB most other scrapers use the `text[/pattern/, 1]` style)